core/src/main/scala/org/http4s/MediaType.scala

```diff
@@ -199,6 +199,7 @@ object MediaType extends Registry {
   val `application/excel`                                                         = app("excel", uncompressible, binary, "xl", "xla", "xlb", "xlc", "xld", "xlk", "xll", "xlm", "xls", "xlt", "xlv", "xlw")
   val `application/font-woff`                                                     = app("font-woff", uncompressible, binary, "woff")
   val `application/gnutar`                                                        = app("gnutar", uncompressible, binary, "tgz")
+  val `application/gzip`                                                          = app("gzip", uncompressible, binary, "gz")
```

Comment:
We should do a resynchronizing with the IANA registry at some point, but :+1: to this first step.